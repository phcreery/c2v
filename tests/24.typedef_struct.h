typedef struct
{
    float x, y;
} ImVec2;

typedef struct ImVec3 ImVec3;
struct ImVec3
{
    float x, y, z;
};

struct ImVec4
{
    float x, y, z, w;
};
typedef struct ImVec4 ImVec4;