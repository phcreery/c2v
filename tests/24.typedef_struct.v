@[translated]
module C:\Users\phcre\Documents\v\c2v\tests

struct ImVec2 { 
	x f32
	y f32
}
struct ImVec3 { 
	x f32
	y f32
	z f32
}
struct ImVec4 { 
	x f32
	y f32
	z f32
	w f32
}
struct ImGuiTextRange { 
	b &i8
	e &i8
}
struct ImVector_ImGuiTextRange { 
	size int
	capacity int
	data &ImGuiTextRange
}
type ImWchar16 = u16
struct ImGuiContext { 
	initialized int
}
