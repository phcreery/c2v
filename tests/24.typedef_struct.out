@[translated]
module tests

struct ImVec2 {
       x f32
       y f32
}
struct ImVec3 {
       x f32
       y f32
       z f32
}
struct ImVec4 {
       x f32
       y f32
       z f32
       w f32
}